egs/mgb2/asr/conf/tuning/train_transformer.yaml

```diff
@@ -0,0 +1,32 @@
+# network architecture
```

Comment:
This is not about the transformer.
Anyway, if this config file is not used, please remove it.

Reply:
done